app/script/assets/AssetCrypto.js

```diff
@@ -88,7 +88,7 @@ z.assets.AssetCrypto = (() => {
   function _generate_random_bytes(length) {
     const randomValues = new Uint32Array(length / 4).map(() => libsodium.getRandomValue());
     const ramdonBytes = new Uint8Array(randomValues.buffer);
```

Comment:
I can see that we already using `camelCase` and I would recommend using them for new variables.. this change can't happen in one go.. so replacing them here and there shouldn't be a problem :)